src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int NUMBER_ZERO = 0;
+    private static final int NUMBER1 = 1;
+    private static final int NUMBER2 = 2;
+    private static final int NUMBER3 = 3;
```

Comment:
Create more understandable name for constant, for example: INDEX_OF_NAME, INDEX_OF_HOURS